src/components/tables/DataTable/table/DataTable.module.scss

```diff
@@ -48,6 +100,35 @@
         }
       }
       
+      thead, tbody {
+        th, td {
+          &.column-sticky {
+            position: sticky;
+            z-index: 1;
+          }
+          
+          &.column-sticky--left {
+            left: 0;
+          }
+          
+          &.column-sticky--right {
+            right: 0;
+          }
+        }
+
+        th.column-sticky,
+        td.column-sticky {
+          background-color: bk.$theme-card-background-default;
+          // For sticky cells we need a solid colour as background,
+          // otherwise during scroll the content scrolled behind sticky cell will appear as overlapping
```

Comment:
Let's use this background color for the entire table, not just the sticky column.